frontend/src/pages/LobbyPage/index.tsx

```diff
@@ -1,21 +1,60 @@
-import React, { useState } from "react";
+import React, { useCallback, useEffect, useState } from "react";
 import { useHistory } from "react-router-dom";
+import debounce from "lodash/debounce";
 import Button from "../../components/Button";
 import PlayerList from "../../components/PlayerList";
 import styles from "./style.module.css";
 import TextField from "../../components/TextField";
 import Dropdown from "../../components/Dropdown";
+import { Settings } from "../../GameRouter";
+import socket from "../../socket";
 
 type Props = {
   gameCode: string;
+  settings?: Settings;
 };
 
-const LobbyPage = ({ gameCode }: Props) => {
+const LobbyPage = ({ gameCode, settings }: Props) => {
```

Comment:
Should add `const { me, host, players } = useContext(PlayersContext)` to this component to disable setting input for non-host and display the number of players in the `<h2>`

Reply:
This is no longer required as of #68 as we can use the players `id` persisted with `use-persisted-state` matched against other player ids in the `PlayersContext`